src/app/leverage/layout.tsx

```diff
@@ -10,9 +10,11 @@ type LayoutProps = {
 export default function Layout({ children }: LayoutProps) {
   return (
     <ProvidersLite>
-      <Header />
-      <main className='bg-[#141E1F]'>{children}</main>
-      <Footer />
+      <div className='dark bg-[#141E1F]'>
+        <Header isDarkMode={true} />
+        <main>{children}</main>
+        <Footer isDarkMode={true} />
```

Comment:
adding a property like this to the components was the easiest way for me right now. Feel free to suggest alternatives.

a more sophisticated solution with hooks, I'd propose only once we get to switching light/dark mode site wide

Reply:
instead of setting `isDarkMode`, could we have the two logos (dark and light) in the header/footer, and then do something like this for them...
light logo: `block dark:hidden`
dark logo: `hidden dark: block`

It gets us away from needing props